api/types/DirectMessageThread.js

```diff
@@ -53,11 +53,17 @@ const DirectMessageThread = /* GraphQL */ `
 	input DirectMessageThreadInput {
 		participants: [ID!]
 		message: DirectMessageContentInput!
-	}
+  }
+
+  input isArchivedDMThreadInput {
+    isArchived: Boolean!
+    threadId: ID!
+  }
 
 	extend type Mutation {
 		createDirectMessageThread(input: DirectMessageThreadInput!): DirectMessageThread
-		setLastSeen(id: ID!): DirectMessageThread
+    setLastSeen(id: ID!): DirectMessageThread
+    changeIsArchivedDirectMessageThread(input: isArchivedDMThreadInput!): DirectMessageThread
```

Comment:
What do you think about calling this `archiveDirectMessageThread` or something like that in order to be more consistent with our existing mutations?

Reply:
Makes sense to me. I will update this on my next commit.